### ISSUE TITLE
[question] Help integrate Centrifugo with Traefik Proxy

### ISSUE THREAD

**Issue by 070hm** [centrifugal/centrifugo#857]
Hi everyone,

Sorry, this question is not directly related to the Centrifugo system, but I am looking for assistance with integrating Centrifugo with Traefik Proxy.
The full problem is described here https://community.traefik.io/t/multiple-port-bindings-for-the-same-host-v3/23563 or https://stackoverflow.com/questions/78742655/multiple-port-bindings-for-the-same-host-v3

**Comment by FZambia** [comment]
Hello, my only guess without debugging is that I see port 20000 from Centrifugo container is exposed but it's not used in the rest of configuration.

**Comment by 070hm** [comment]
> Hello, my only guess without debugging is that I see port 20000 from Centrifugo container is exposed but it's not used in the rest of configuration.

Yes, I exposed port 20000 for public access and port 10000 for private access. I want to use port 20000 from Traefik to map to port 10000. The problem is that the container exposes two ports: 8000:8000 for REST API and 20000:10000 for gRPC. I want Traefik to handle them. I tried some configurations in Traefik, but they are not working for me.

By the way, thanks for the feedback. It's nice of you to reply to some questions not related to the project 🙏
